runtime/libtapasco/src/device.rs

```diff
@@ -411,7 +426,7 @@ impl Device {
                     ),
                 }));
             } else {
-                trace!("Using SVM...");
+                println!("Using SVM...");
```

Comment:
I think we should stick to our tracing macros like ```trace!```, ```info!``` and so on, what is the reason that you changed it to ```println!``` here?

Reply:
this is probably a remnant from when i was unaware of the RUST_LOG=trace variable and changed all occurences to println! I will change it back.